examples/text_classification/predict.py

```diff
@@ -6,6 +6,18 @@
 
 
 def predict(text, model, dictionary):
```

Comment:
dictionary here is a misnomer on my end since we ended up standardizing on the legacy "vocab"